### DISCUSSION TITLE
Load font to use directly in ImGui components

### DISCUSSION THREAD

**Discussion by iWanheda** [ocornut/imgui#5141] [New Users Build/Link/Run issues ONLY!]
Hey!

I can't load font between framestart and frameend, and I'm trying a theme that uses pushfont and popfont directly but I have no way of loading the font without either it being null or I crash because I can't load before something, it's really bugging me out.

Thanks!

**Comment by rokups** [comment]
Fonts may not be modified during lifetime of a frame. If you wish to switch a font, then do it after frame ended and before next one started.
